vhd/packer/configure-windows-vhd.ps1

```diff
@@ -81,9 +81,9 @@ function Get-FilesToCacheOnVHD
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.6/windowszip/v1.16.6-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.7/windowszip/v1.16.7-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.8/windowszip/v1.16.8-1int.zip",
-            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.2/windowszip/v1.17.2-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.3/windowszip/v1.17.3-1int.zip",

```

Comment:
I think we need to take one more version away here

Reply:
AFAICT we have been keeping three patch versions around consistently in the Windows VHD. Intentional, or sloppy? @marosset?

Reply:
Intentional. So I think this PR is ready as-is.